tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -43,16 +43,16 @@ def task(event, context):  # pylint: disable-msg=unused-argument
     LOGGER.debug("event: {event}", event=event)
     try:
         config = event["config"]
-        exclude_file_types = config.get(CONFIG_EXCLUDE_FILE_TYPES_KEY, None)
+        exclude_file_types = config.get(CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS, None)
         if exclude_file_types is None:
             exclude_file_types = []
         if len(exclude_file_types) == 0:
             LOGGER.debug(
-                f"The configuration list {CONFIG_EXCLUDE_FILE_TYPES_KEY} is empty."
+                f"The configuration list {CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS} is empty."
```

Comment:
Possibly change to `CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS_KEY` to match naming convention